Cargo.toml

```diff
@@ -1,19 +1,18 @@
-[package]
-name = "rustpython"
-version = "0.2.0"
-authors = ["RustPython Team"]
-edition = "2021"
-rust-version = "1.67.1"
-description = "A python interpreter written in rust."
-repository = "https://github.com/RustPython/RustPython"
-license = "MIT"
-include = ["LICENSE", "Cargo.toml", "src/**/*.rs"]
+#[package]
```

Comment:
I removed all unused crates from the workspace setup rather than deleting them. This should make it easier for us to sync with upstream (rather than dealing with delete merge conflicts all the time).